### DISCUSSION TITLE
Adding enabled or pause option in useQuery

### DISCUSSION THREAD

**Discussion by sharmapukar217** [rakkasjs/rakkasjs#84] [Q&A]
While this might not be needed for everyone, while developing my app, I had to run query only when some condition meets, so i was thinking of an option to pause query.

**Comment by cyco130** [comment]
Passing `undefined` as the key used to work but I may have broken it since, there are no tests and I don't use it myself. I can't try, I'm away from my computer. 

Let's keep this open, I'll add the `enabled` option though.

**Comment by cyco130** [comment]
This has been implemented [sometime ago](https://github.com/rakkasjs/rakkasjs/pull/105). So we can close this one :)
